Sources/Interaction/Style/InteractorStyleManipulatorRotateWorld/example/index.js

```diff
@@ -0,0 +1,192 @@
+
+function extendInteractorStyle(publicAPI, model, initialValues = {}) {
+  Object.assign(model, initialValues);
+
+  // Extend the base InteractorStyleManipulator
+  vtkInteractorStyleManipulator.extend(publicAPI, model, initialValues);
+
+  // Add our custom methods
+  vtkInteractorStyleRotateWorld(publicAPI, model);
+}
+
+const newInteractorStyleRotateWorld = macro.newInstance(
+  extendInteractorStyle,
+  'vtkInteractorStyleRotateWorld'
+);
+
+// ----------------------------------------------------------------------------
+// Standard rendering code setup
+// ----------------------------------------------------------------------------
+
+const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance();
+const renderer = fullScreenRenderer.getRenderer();
+const renderWindow = fullScreenRenderer.getRenderWindow();
+
+// ----------------------------------------------------------------------------
+// Example code
+// ----------------------------------------------------------------------------
+
+// Use our custom interactor style
+const interactorStyle = newInteractorStyleRotateWorld();
+fullScreenRenderer.getInteractor().setInteractorStyle(interactorStyle);
+
+// ----------------------------------------------------------------------------
+// Create cone (main object)
+// ----------------------------------------------------------------------------
+
+const coneSource = vtkConeSource.newInstance({ height: 1.0 });
+const coneMapper = vtkMapper.newInstance();
+coneMapper.setInputConnection(coneSource.getOutputPort());
+
+const coneActor = vtkActor.newInstance();
+coneActor.setMapper(coneMapper);
+coneActor.getProperty().setColor(0.5, 0.5, 1.0);
+
+// ----------------------------------------------------------------------------
+// Add actors and setup camera
+// ----------------------------------------------------------------------------
+
+renderer.addActor(coneActor);
+renderer.resetCamera();
+
+// Set the center of rotation (this will stay fixed due to our overrides)
+const coneCenter = [0, 0, 0];
+interactorStyle.setCenterOfRotation(coneCenter);
+
+// ----------------------------------------------------------------------------
+// Setup manipulators
+// ----------------------------------------------------------------------------
+
+// Rotate with left button
+const rotateManipulator =
+  vtkMouseCameraTrackballRotateManipulator.newInstance();
+rotateManipulator.setButton(1); // Left button
+interactorStyle.addMouseManipulator(rotateManipulator);
+
+// Zoom with middle button
+const middleZoomManipulator =
+  vtkMouseCameraTrackballZoomManipulator.newInstance();
+middleZoomManipulator.setButton(2); // Middle button
+interactorStyle.addMouseManipulator(middleZoomManipulator);
+
+// Also allow pan with shift + left button
+const shiftPanManipulator = vtkMouseCameraTrackballPanManipulator.newInstance();
+shiftPanManipulator.setButton(1);
+shiftPanManipulator.setShift(true);
```

Comment:
This should have an option to enable or disable center of rotation modification. The default should remain the way it was before.